MiniBootcamp/MiniBootcamp/Utilities/NetworkConstants.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  NetworkConstants.swift
+//  MiniBootcamp
+//
+//  Created by Edgar Solis on 27/05/23.
+//
+
+import Foundation
+
+struct NetworkConstants {
+    static let tweetsURL = "https://gist.githubusercontent.com/ferdelarosa-wz/0c73ab5311c845fb7dfac4b62ab6c652/raw/6a39cffe68d87f1613f222372c62bd4e89ad06fa/tweets.json"
```

Comment:
After declaring either a class, a struct or an enum, you have to leave and empty line.
```suggestion
struct NetworkConstants {

    static let tweetsURL = "https://gist.githubusercontent.com/ferdelarosa-wz/0c73ab5311c845fb7dfac4b62ab6c652/raw/6a39cffe68d87f1613f222372c62bd4e89ad06fa/tweets.json"
```

Also. it would be great to add an inline comment.